spec/game_start_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+require_relative '../dependencies'
+
+
+RSpec.describe GameStart do
+  let(:game_class) { Class.new { extend GameStart } }
+
+  describe '.check_numbers' do
+    let(:examples) { YAML.load_file('./spec/examples.yml') }
```

Comment:
move to spec/fixtures/

Reply:
I haven't found how to do it. Basically it redirects to using fixtures in Rails. Can you help me?

Reply:
just move `examples.yml` file from `spec/` to `spec/fixtures/`